build.gradle

```diff
@@ -1,7 +1,7 @@
 buildscript {
   ext.versions = [
-    'kotlin': '1.3.70',
-    'serializationRuntime': '0.20.0',
+    'kotlin': '1.4.0-rc',
+    'serializationRuntime': '1.0-M1-1.4.0-rc',
```

Comment:
@NitroG42 When using Kotlin 1.4.0 I have used org.jetbrains.kotlinx:kotlinx-serialization-core:1.0.0-RC as a dependency. (runtime jar was renamed to core).
When using your fork, I also had to exclude org.jetbrains.kotlinx:kotlinx-serialization-runtime and org.jetbrains.kotlinx:kotlinx-serialization-runtime-jvm from being included to fix duplicate class definitions.

Reply:
I'm probably going to rewrite the pull requests for the latest change, and I also saw that 1.4.0, while out, still don't have a blog post announcement (which probably means that it's not officialy out). I wonder if they are waiting for serialization (and maybe others kotlinx library) to be released in 1.0.

Reply:
Nope! Looks like you just had to wait a few hours:
https://blog.jetbrains.com/kotlin/2020/08/kotlin-1-4-released-with-a-focus-on-quality-and-performance/